templates/component/name.component.js

```diff
@@ -5,6 +5,7 @@ angular.module('<%= scriptAppName %>')
     template: '<h1>Hello {{ $ctrl.message }}</h1>',
     bindings: { message: '<' },
     controller: function() {
-      this.message = 'World'
+      var vm = this;
+      vm.message = 'World';
```

Comment:
not inlining the controller function is good, but creating a new `vm = this` here is bad IMO. It's just unnecessarily turning a 1-line controller into a 2-line controller. Now, if your controller was much larger it could make sense to store  the class `this` in a var, but not here.